backend/src/main/kotlin/com/hootsuite/hermes/slack/SlashCommandHandler.kt

```diff
@@ -26,7 +26,7 @@ object SlashCommandHandler {
                 }
                 if (parameters.size == 1) {
                     DatabaseUtils.createOrUpdateUserByGithubName(User(parameters.first(), slashCommand.username, team))
-                    if (DatabaseUtils.getTeamOrNull(team) != null) {
+                    if (DatabaseUtils.getTeamOrNull(team)) {
```

Comment:
Is this a new language feature I don't know about?

Reply:
changed the method signature without updating the name of the method, updated